### ISSUE TITLE
New folder is added on top of existent one

### ISSUE THREAD

**Issue by rainer2208** [spheras/desktopfolder#241]
When adding a new folder, it is created in the top left corner, on top of a folder already there. It would be better to have it created in an empty space.

**Comment by heetu** [comment]
Yup Sir i have a same problem occur.(Behaviour=Align to grid(on))

**Comment by spheras** [comment]
Hi all, Maybe are you running the last released version on elementary OS?  I say that because this was solved in the current branch (but not released yet). Can you confirm if this behaviour is happening in the master branch (I can't reproduce)

**Comment by fossfreedom** [comment]
hmm... this one is a bit odd.  Kind of hit and miss.

killed DesktopFolder process
I deleted everything from ~/Desktop and the .desktopfolder file

Restarted DesktopFolder

Right click create new folder and name it "test"

Right click create new folder and name it "test" <--- that is an error - it appears it creates a second "test folder" but underneath it is and "untitled folder"

Right click create a new file ... call it test ... you see the second test folder being renamed

Right click create a new folder ... call it test ... it appears over the top of the original test folder

[image.png]

Picture shows the two test folders

On Thu, 17 Jan 2019 at 09:36, José Amuedo Salmerón <notifications@github.com<mailto:notifications@github.com>> wrote:

Hi all, Maybe are you running the last released version on elementary OS? I say that because this was solved in the current branch (but not released yet). Can you confirm if this behaviour is happening in the master branch (I can't reproduce)

**Comment by spheras** [comment]
:upside_down_face: well seen @fossfreedom. Not sure if this is the same problem reported originally, but very good issue :smile:

**Comment by spheras** [comment]
I think this is solved, I can't reproduce in the current master branch. Please, reopen if necessary. thanks!!

**Comment by rainer2208** [comment]
> Hi all, Maybe are you running the last released version on elementary OS? I say that because this was solved in the current branch (but not released yet). Can you confirm if this behaviour is happening in the master branch (I can't reproduce)

@spheras , I am using a fresh install of UB, version 18.10, everything updated ....

**Comment by heetu** [comment]
> Hi all, Maybe are you running the last released version on elementary OS? I say that because this was solved in the current branch (but not released yet). Can you confirm if this behaviour is happening in the master branch (I can't reproduce)

@spheras ,In New updates i thing problem was solved.Thank You
